scrap-test.sql

```diff
@@ -1,19 +1,14 @@
 \c nc_games_test
 
```

Comment:
This also should ideally not be in this commit. 
Try to avoid git add . , as it adds all files.